controller/src/beerocks/master/tasks/dynamic_channel_selection_task.h

```diff
@@ -20,6 +20,7 @@
 // Helper MACROs for Enum/String generation
 #define GENERATE_ENUM(ENUM) ENUM,
 #define GENERATE_STRING(STRING) #STRING,
+#define NO_SCAN_IN_PROGRESS -1
```

Comment:
No need for this define, we use `-1` for invalid task id.
Maybe should add an `INVALID_TASK_ID` to the base class (but it's not part of this PR anyway).

Reply:
removed the define